ci/run-tests.sh

```diff
@@ -22,6 +22,17 @@ pushd "$(dirname "$0")"
 
     pushd "${UNREAL_PATH}"
         UNREAL_EDITOR_PATH="Engine/Binaries/Mac/UE4Editor.app/Contents/MacOS/UE4Editor"
+
+        echo "Generating test maps for testing project"
+        "${UNREAL_EDITOR_PATH}" \
+                "${UPROJECT_PATH}" \
+                -SkipShaderCompile \
```

Comment:
By the way, this doesn't seem to work, still says compiling shaders. Been this way for a while

Reply:
`-SkipShaderCompile` needs to be added as `-additionalcookeroptions=-"-SkipShaderCompile`"

Reply:
You may get away with `-additionalcookeroptions=-SkipShaderCompile" but not sure. Try both

Reply:
Any news on this?

Reply:
Ah, I think I decided at some point to not tackle it with this PR.
Ideally what I would want is for shaders to not be compiled at all, since graphics are not important for us.
However this seems to suggets they would not be compiled just for the cooking process, which is only 1 of the started processes. Can create a separate ticket to look at this.